configurations/main-footer.development.config.json

```diff
@@ -1,180 +1,190 @@
+            "url": "https://eosc-portal.eu/eosc-in-practice/use-cases"
+          },
+          {
+            "label": "Submit Use Case",
+            "url": "https://eosc-portal.eu/eosc-in-practice/open-call"
+          },
+          {
+            "label": "Industry Use Case",
+            "url": "https://eosc-portal.eu/industry-use-cases"
+          }
+        ]
+      },
+      {
+        "label": "For providers",
+        "url": "https://eosc-portal.eu/for-providers",
+        "navBtns": [
+          {
+            "label": "Providers Documentation",
+            "url": "https://eosc-portal.eu/providers-documentation"
+          },
+          {
+            "label": "Promote your services",
+            "url": "#"
+          }
+        ]
+      }
+    ],
+    [
+      {
+        "label": "Services & Resources",
+        "url": "https://eosc-portal.eu/services-resources",
+        "navBtns": [
+          {
+            "label": "Sharing & Discovery",
+            "url": "https://marketplace.eosc-portal.eu/categories/sharing-discovery"
+          },
+          {
+            "label": "Training & Support",
+            "url": "https://marketplace.eosc-portal.eu/categories/training-support"
+          },
+          {
+            "label": "Security & Operation",
+            "url": "https://marketplace.eosc-portal.eu/categories/security-operations"
+          },
+          {
+            "label": "Processing & Analysis",
+            "url": "https://marketplace.eosc-portal.eu/categories/processing-analysis"
+          },
+          {
+            "label": "Access physical & einfrastructures",
+            "url": "https://marketplace.eosc-portal.eu/categories/access-physical-einfrastructures"
+          },
+          {
+            "label": "Aggregators & Integrators",
+            "url": "https://marketplace.eosc-portal.eu/categories/aggregators-integrators"
+          },
+          {
+            "label": "Other",
+            "url": "https://marketplace.eosc-portal.eu/categories/other"
+          },
+          {
+            "label": "Help Desk",
+            "url": "https://eosc-portal.eu/helpdesk"
+          }
+        ]
+      }
+    ],
+    [
+      {
+        "label": "Media",
+        "navBtns": [
+          {
+            "label": "News",
+            "url": "https://eosc-portal.eu/media/news"
+          },
+          {
+            "label": "Forthcoming events",
+            "url": "https://eosc-portal.eu/media/events"
+          },
+          {
+            "label": "Past events",
+            "url": "https://eosc-portal.eu/media/past-events"
+          },
+          {
+            "label": "Submit your News/Event",
+            "url": "https://eosc-portal.eu/contribute-to-eosc"
+          },
+          {
+            "label": "Community",
+            "url": "https://eosc-portal.eu/documents/community"
+          },
+          {
+            "label": "Newsletters",
+            "url": "#"
```

Comment:
Once again, is it correct? In other places it is: `"https://eosc-portal.eu/newsletters"`